maps/southern_cross/southern_cross_areas.dm

```diff
@@ -1176,8 +1176,9 @@ z
 	icon_state = "docking_hallway"
 
 /area/hallway/secondary/seconddeck/research_medical
-	name = "Research Medical Hallway"
+	name = "Research Medical Maintenance"
 	icon_state = "hallS"
+	flags = RAD_SHIELDED //CHOMPedit: Adding rad proofing to new maints
```

Comment:
Instead of adding rad proof, change the area in the map to "/area/maintenance/research_medical". This will prevent vines, viruses and any other kind of stuff to spawn in these areas, and make them rad proof

Reply:
This would work fine I reckon, but also don't forget to remove one of the APCs as well, as this would combine two areas into one leaving the combined area with duplicate APCs

Reply:
Oh, yeah mentioned this on discord but my strongDMM keeps crashing whenever I attempt to open anything with it.
Putting it here for easier relevant viewing